ResearchKit/Common/ORKFormStepViewController.m

```diff
@@ -825,6 +828,12 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                         [formCell setExpectedLayoutWidth:self.tableView.bounds.size.width];
                         formCell.selectionStyle = UITableViewCellSelectionStyleNone;
                         formCell.defaultAnswer = _savedDefaults[formItem.identifier];
+                        if ([class isSubclassOfClass:[ORKFormItemConfirmTextCell class]]) {
```

Comment:
`setSavedAnswers` is available on `ORKFormItemCell`, do we need this check here?